test_cli.py

```diff
@@ -279,33 +170,21 @@ def wait_until_substring(fp, substr):
             return
 
 
-class Colorizer(object):
-    RED = "\033[31;1m"
-    GREEN = "\033[32;1m"
-    YELLOW = "\033[33;1m"
-    CYAN = "\033[36;1m"
-    RESET = "\033[0m"
-    NEWLINE = "\n"
-
-    @classmethod
-    def _colorize(cls, string, color):
-        return getattr(cls, color) + string + cls.RESET + cls.NEWLINE
-
-    @classmethod
-    def red(cls, string):
-        return cls._colorize(string, "RED")
-
-    @classmethod
-    def green(cls, string):
-        return cls._colorize(string, "GREEN")
-
-    @classmethod
-    def yellow(cls, string):
-        return cls._colorize(string, "YELLOW")
-
-    @classmethod
-    def cyan(cls, string):
-        return cls._colorize(string, "CYAN")
+def _run_command(
```

Comment:
Would it be better just to get rid of this function and call `run_command` from all the tests instead?

Reply:
This is because of the `force_subprocess` check we had previously.  I don't want to check or have to pass in the `cl` instance everytime I call the `run_command`.